src/components/calculator.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import '../calc.css';
 
-export default class Calculator extends React.PureComponent {
-  render() {
+class Calculator extends React.Component {
+  constructor() {
+    super();
+    this.MapCalculator = this.MapCalculator.bind(this);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
## [OPTIONAL]
- [ ] Kindly solve linters instead of disabling them as they help you structure well your code and prevent future bugs.